### PR TITLE
Release of a game

### PR DESCRIPTION
**Pull request by tarasavage**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1 +1,31 @@
...
+      || userInput.split('').some((e, i, arr) => arr.indexOf(e) !== i))) {
+      console.log('Wrong input!');
+      numberIsCorrect = false;
+      game(randomInput);
+    }
+
+    const animals = compareNumbers(randomInput, userInput);
+
+    if (animals.bulls === 4) {
+      console.log('Congratulations! You won!');
```
Comment (polosanya) [inline src/app.js:24]: Let's try not to use console log. Just return the string with proper message instead.
Fix all cases

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's try to pass the tests)

File: src/app.js [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+const random = generateNumber();
+
+(function game(randomInput) {
+  terminal.question('Please, enter the number: ', (userInput) => {
+    let numberIsCorrect = true;
+
+    if ((userInput.length !== 4
+      || userInput.split('').some(num => isNaN(+num))
+      || userInput.split('').some((e, i, arr) => arr.indexOf(e) !== i))) {
```
Comment (Bohdan-Maksymiuk) [inline src/app.js:15]: lets split these conditions into separate variables and then use here

File: src/calculateBullsAndCows.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  const digitsRandom = `${random}`.split('');
+  const digitsInput = `${input}`.split('');
+
+  for (let i = 0; i < digitsRandom.length; i++) {
+    if (digitsInput[i] === digitsRandom[i]) {
+      count.bulls++;
+    } else if (digitsRandom.includes(digitsInput[i])) {
+      count.cows++;
+    }
```
Comment (Bohdan-Maksymiuk) [inline src/calculateBullsAndCows.js:17]: avoid such nesting `else if` conditions
This code is hard to read and maintain

File: src/app.js [hunk 4]
```diff
@@ -1 +1,34 @@
...
+
+    if (animals.bulls === 4) {
+      terminal.close();
+
+      return 'Congratulations! You won!';
+    } else if (numberIsCorrect) {
+      terminal.close();
+
+      return `Bulls: ${animals.bulls} Cows: ${animals.cows}`;
+    }
```
Comment (Bohdan-Maksymiuk) [inline src/app.js:32]: avoid such nesting `else if` conditions
This code is hard to read and maintain

**Review by nazarmatsevych — APPROVED** [review]
Nicely done!

**Pull request closed without merging**